jobs/upcoming.js

```diff
@@ -35,8 +35,15 @@ module.exports = async () => {
     });
 
     // Past launches needed to set new flight number order
-    const upcoming = rawLaunches.docs.filter((doc) => doc.upcoming === true);
-    const past = rawLaunches.docs.filter((doc) => doc.upcoming === false);
+    const [upcoming, past] = rawLaunches.docs.reduce((curr, acc) => {
```

Comment:
While this might be more performant, I think using two filters keeps things readable and easier to understand.